src/components/Tool/FixMailingAddresses/Contact.tsx

```diff
@@ -233,7 +235,9 @@ const Contact: React.FC<Props> = ({
                       <strong>{t('Source')}: </strong>
                     </Typography>
                   </Hidden>
-                  <Typography display="inline">MPDX</Typography>
+                  <Typography display="inline">
```

Comment:
@dr-bizz this should be removed altogether, not changed to appName. "MPDX" is still showing up across from the "Add Address" button.